portfolio/src/main/webapp/photography.html

```diff
@@ -4,22 +4,39 @@
     <meta charset="UTF-8">
     <title>Photography</title>
     <link rel="stylesheet" href="style.css">
+        <style>
+        h1 {text-align: center;}
+p {text-align: center;}
+div {text-align: center;} 
+h2 {text-align: center;}
+h3 {text-align: center;}  
+
+    </style>
   </head>
   <body>
-    <div id="content">
-      <h1>Photography</h1>
-      <h2>Why I love taking pictures</h2>
+      <div class="bg-image img1"></div>
+<div class="bg-image-photo img2"></div>
+<div class="bg-image-photo img3"></div>
+<div class="bg-image-photo img4"></div>
+<div class="bg-image-photo img5"></div>
+<div class="bg-image-photo img6"></div>
+    <div class="bg-text-photo">
+      <h1 class="w3-cinzel">PHOTOGRAPHY</h1>
+      </div> <div>
```

Comment:
the indentation seems off here

In general, can you check that the file matches the style guide?

Reply:
Done